kickstarts/partials/post/ovirt_ansible.ks.erb

```diff
@@ -0,0 +1,5 @@
+# Add oVirt repository
+yum install -y https://resources.ovirt.org/pub/yum-repo/ovirt-release42.rpm
```

Comment:
Just please don't forget that oVirt is upstream only repository, which is fine for ManageIQ. But for CFME we should be using downstream builds from RHV erratas.

Reply:
yup, downstream will be taken care of differently!

Reply:
`https://resources.ovirt.org/pub/yum-repo` is called `yum-repo` ... and it isn't a repo? :laughing: 

@mwperina is this really the case?

Also, I can't seem to download the rpm ... or get to https://resources.ovirt.org/ at all ... is this website even live?

```
$ wget https://resources.ovirt.org/pub/yum-repo/ovirt-release42.rpm
--2018-09-24 11:28:52--  https://resources.ovirt.org/pub/yum-repo/ovirt-release42.rpm
Resolving resources.ovirt.org (resources.ovirt.org)... 66.187.230.28
Connecting to resources.ovirt.org (resources.ovirt.org)|66.187.230.28|:443... connected.
GnuTLS: Error in the pull function.
Unable to establish SSL connection.
```
:point_up: took a bit over a minute

```
sudo yum install https://resources.ovirt.org/pub/yum-repo/ovirt-release42.org
[sudo] password for ncarboni: 
Last metadata expiration check: 2:14:20 ago on Mon 24 Sep 2018 09:16:55 AM EDT.
[MIRROR] ovirt-release42.org: Curl error (28): Timeout was reached for https://resources.ovirt.org/pub/yum-repo/ovirt-release42.org [Operation timed out after 30001 milliseconds with 0 out of 0 bytes received] 
[FAILED] ovirt-release42.org: Curl error (28): Timeout was reached for https://resources.ovirt.org/pub/yum-repo/ovirt-release42.org [Operation timed out after 30001 milliseconds with 0 out of 0 bytes received] 
Curl error (28): Timeout was reached for https://resources.ovirt.org/pub/yum-repo/ovirt-release42.org [Operation timed out after 30001 milliseconds with 0 out of 0 bytes received]
```

As it stands it looks like this might just break the build or in the best case scenario just wouldn't work ...

Reply:
Maybe a temporary network issue? I did a full build with this change on Friday, and everything worked ok...

Reply:
Could be. I was trying to pull the RPM to get the contents of the repo it provides to see if we could do something more clever here, but I still can't get to it.

Reply:
The RPM is official supported way how to setup on a computer the repositories required for oVirt installation. Installing ovirt-release42 is easiest way how to get installed latest version of python SDK and ansible roles.

There was an issues with HTTP server, I've ping our administrators and the issue was resolved.